### DISCUSSION TITLE
Baresip UA as a SIP Trunk

### DISCUSSION THREAD

**Discussion by AviWA** [juha-h/baresip-studio#445] [Ideas]
I am trying to make sip client UA as sip trunk to maange the audio in android device. But I dont know how to do that. Please help me to achieve this.

Thanks, Regards.

**Comment by juha-h** [comment]
AviWA writes:

> I am trying to make sip client UA as sip trunk to maange the audio in
> android device. But I dont know how to do that. Please help me to
> achieve this.

Sorry, but I don't have resources for other Android projects.

**Reply by AviWA** [reply]
Just give me any guidance to make this happen. I am new to sip development. I am not able to register to sip trunk in asterisk or freepbx. so I am trying to do this.
